### PR TITLE
fix defs table update

### PR DESCRIPTION
**Pull request by beyang**
Fix the update of defs table to make the update repo-wide, as opposed to one source unit at a time (which was overwriting previous source unit data).

### CODE REVIEW

File: services/httpapi/srclib_import.go [hunk 1]
```diff
@@ -100,6 +100,14 @@ func serveSrclibImport(w http.ResponseWriter, r *http.Request) (err error) {
 		return fmt.Errorf("srclib import of %s failed: %s", repo.URI, err)
 	}
 
+	// Update defs table in DB
+	if _, err := cl.Defs.RefreshIndex(ctx, &sourcegraph.DefsRefreshIndexOp{
+		Repo:     repoRev.Repo,
+		CommitID: repoRev.CommitID,
+	}); err != nil {
+		return err
+	}
```
Comment (shurcooL) [inline services/httpapi/srclib_import.go]: Do we want this to happen for forks too?

**Comment by shurcooL** [comment]
Not seeing any issues in the code.

File: services/httpapi/srclib_import.go (hunk 1 above)
Comment (beyang) [inline services/httpapi/srclib_import.go]: Yes, it should happen for any repository that we build.

**Comment by beyang** [comment]
Thanks for the quick review.

**Pull request merged**